init-tools.cmd

```diff
@@ -0,0 +1,79 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+:CheckOS
+IF "%PROCESSOR_ARCHITECTURE%"=="x86" (set bit=x86) else (set bit=x64)
```

Comment:
Because we need do more than just build managed code in our repo.  For example, we crossgen managed code. Crossgen'ing needs a specific RID to work, i.e. we need the x86 RID.  The way our CrossGen tool figures out which RID to use is by looking at the architecture of the current process (i.e. the "build" process):

```C#
        private static string GetArch()
        {
#if NET451
            return Environment.Is64BitProcess ? "x64" : "x86";
#else
            return IntPtr.Size == 8 ? "x64" : "x86";
#endif
        }
```

And then later:

```
        private string GetCoreCLRRid()
        {
            string rid = null;
            if (CurrentPlatform.IsWindows)
            {
                var arch = RuntimeEnvironment.RuntimeArchitecture;
                rid = $"win7-{arch}";
            }
```

which is called by:

```
        private string GetCrossGenPackagePathForVersion()
        {
            string coreclrRid = GetCoreCLRRid();

            if (coreclrRid == null)
            {
                return null;
            }

            string packageId = $"runtime.{coreclrRid}.Microsoft.NETCore.Runtime.CoreCLR";

            return Path.Combine(
                Dirs.NuGetPackages,
                packageId,
                _coreClrVersion);
        }
```

Thus, the process needs to be x86 when we are building the x86 CLI.